base-template.sh

```diff
@@ -540,9 +598,24 @@ update_shared_hooks_if_appropriate() {
                     echo "$PULL_OUTPUT" >&2
                 fi
             else
-                echo "* Retrieving shared hooks from: $SHARED_REPO"
+                echo "* Retrieving shared hooks from: $SHARED_REPO_CLONE_URL"
+
                 [ -d "$SHARED_ROOT" ] && rm -rf "$SHARED_ROOT"
-                CLONE_OUTPUT=$(git -c core.hooksPath=/dev/null clone "$SHARED_REPO" "$SHARED_ROOT" 2>&1)
+
+                if [ -n "$SHARED_REPO_CLONE_BRANCH" ]; then
+                    CLONE_OUTPUT=$(git -c core.hooksPath=/dev/null clone \
+                        --single-branch \
+                        --branch "$SHARED_REPO_CLONE_BRANCH" \
+                        "$SHARED_REPO_CLONE_URL" \
+                        "$SHARED_ROOT" 2>&1)
+                else
+                    # shellcheck disable=SC2090,SC2086
```

Comment:
Why do we need to disable these? (if we don't have to above)

Reply:
Ups left over